samples/silly-sample/src/sillycode.cpp

```diff
@@ -260,3 +260,8 @@ int i;
 for (i = 0; i < max; ++i); // bug waiting to happen
 if (i == j)
     return i;
+
+void cast_check() {
+  int16_t a = 0;
+  uint64_t b = (int64_t)a;
+}
```

Comment:
Not sure about this since we already have a C-style cast above

Reply:
But this is the first one that uses a fixed-width specifier. Your call whether you want this in the integration test or just the unit tests.

Reply:
This is the first fixed-width type though. Happy to remove it though if you think it's overkill in the integration test scope.

Reply:
We don't and never errored on types with _t. If you really wanted to, you could change the cast above to a [fixed-width floating-point](https://en.cppreference.com/w/cpp/types/floating-point) cast.

Reply:
Maybe the point of the test case was misunderstood.  The point was to get a line of the form `anytype var = (int64_t)foo;` in there, but I can see how the type mismatch on 266 confused things.  Since there was lukewarm reception for an integration test for this subcase anyway, I just removed this clitest case and we can stand on the strength of the unit test cases.